<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerTableBackgroundDialog</class>
 <widget class="QDialog" name="QgsComposerTableBackgroundDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table Background Colors</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="12" column="2">
    <widget class="QgsColorButtonV2" name="mLastRowColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="mOddRowsCheckBox">
     <property name="text">
      <string>Odd rows</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QCheckBox" name="mHeaderRowCheckBox">
     <property name="text">
      <string>Header row</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check options to enable shading for matching cells. Options lower in this list will take precedence over higher options. Eg, if both &amp;quot;&lt;span style=&quot; font-style:italic;&quot;&gt;First row&lt;/span&gt;&amp;quot; and &amp;quot;&lt;span style=&quot; font-style:italic;&quot;&gt;Odd rows&lt;/span&gt;&amp;quot; are checked, the cells in the first row will be shaded using the color specified for &amp;quot;&lt;span style=&quot; font-style:italic;&quot;&gt;First row&lt;/span&gt;&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QCheckBox" name="mLastColumnCheckBox">
     <property name="text">
      <string>Last column</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="mOddColumnsCheckBox">
     <property name="text">
      <string>Odd columns</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="mEvenRowsCheckBox">
     <property name="text">
      <string>Even rows</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="mEvenColumnsCheckBox">
     <property name="text">
      <string>Even columns</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QgsColorButtonV2" name="mOddColumnsColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QgsColorButtonV2" name="mEvenColumnsColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QgsColorButtonV2" name="mFirstRowColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QgsColorButtonV2" name="mLastColumnColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QCheckBox" name="mFirstRowCheckBox">
     <property name="text">
      <string>First row</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="2">
    <widget class="QgsColorButtonV2" name="mFirstColumnColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QgsColorButtonV2" name="mDefaultColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QgsColorButtonV2" name="mHeaderRowColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="mFirstColumnCheckBox">
     <property name="text">
      <string>First column</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QCheckBox" name="mLastRowCheckBox">
     <property name="text">
      <string>Last row</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QgsColorButtonV2" name="mEvenRowsColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Default cell background</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QgsColorButtonV2" name="mOddRowsColorButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgscolorbuttonv2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mDefaultColorButton</tabstop>
  <tabstop>mOddColumnsCheckBox</tabstop>
  <tabstop>mOddColumnsColorButton</tabstop>
  <tabstop>mEvenColumnsCheckBox</tabstop>
  <tabstop>mEvenColumnsColorButton</tabstop>
  <tabstop>mOddRowsCheckBox</tabstop>
  <tabstop>mOddRowsColorButton</tabstop>
  <tabstop>mEvenRowsCheckBox</tabstop>
  <tabstop>mEvenRowsColorButton</tabstop>
  <tabstop>mFirstColumnCheckBox</tabstop>
  <tabstop>mFirstColumnColorButton</tabstop>
  <tabstop>mLastColumnCheckBox</tabstop>
  <tabstop>mLastColumnColorButton</tabstop>
  <tabstop>mHeaderRowCheckBox</tabstop>
  <tabstop>mHeaderRowColorButton</tabstop>
  <tabstop>mFirstRowCheckBox</tabstop>
  <tabstop>mFirstRowColorButton</tabstop>
  <tabstop>mLastRowCheckBox</tabstop>
  <tabstop>mLastRowColorButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mOddColumnsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mOddColumnsColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mEvenColumnsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mEvenColumnsColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mOddRowsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mOddRowsColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mEvenRowsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mEvenRowsColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFirstColumnCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFirstColumnColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mLastColumnCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mLastColumnColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mHeaderRowCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mHeaderRowColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mFirstRowCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mFirstRowColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mLastRowCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mLastRowColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
